File: results/py/DISTRIBUTING_ITEMS_PERSON_CANNOT_TAKE_TWO_ITEMS_TYPE_1.cpp
bool f_filled ( int * arr , int n , int k ) {
  std :: map < int , int > mp ;
  for ( int i = 0 ;
  i < n ;
  i ++ ) mp [ arr [ i ] ] ++ ;
  for ( std :: map < int , int > :: iterator it = mp . begin ( ) ;
  it != mp . end ( ) ;
  it ++ ) if ( it -> second > 2 * k ) return false ;
  return true ;
}

File: results/java_codex/DISTRIBUTING_ITEMS_PERSON_CANNOT_TAKE_TWO_ITEMS_TYPE_1.java
bool checkCount(int arr[], int n, int k) {
        unordered_map<int, int> hash;
        for (int i = 0; i < n; i++) {
            if (hash.find(arr[i]) == hash.end()) hash[arr[i]] = 0;
            hash[arr[i]]++;
        }
        for (auto x : hash) if (x.second > 2 * k) return false;
        return true;
    }